bonfire/Modules/Tools/Views/logs.php

```diff
@@ -0,0 +1,64 @@
+<?php $this->extend('master') ?>
+
+<?php $this->section('main') ?>
+    <x-page-head>
+        <h2>Logs</h2>
+    </x-page-head>
+
+    <x-admin-box>
+
+      <?php if (count($logs)) { ?>
+
+        <form action="<?= site_url(ADMIN_AREA . 'tools/delete-log'); ?>" method="post">
+            <?= csrf_field() ?>
+
+        <div class="table-responsive">
+          <table class="table table-hover logs" cellspacing="0" width="100%" >
+              <thead>
+                  <tr>
+                      <th class="column-check"><input class="check-all" type="checkbox" /></th>
+                      <th class='date'><?= lang('Logs.date'); ?></th>
+                      <th><?= lang('Logs.file'); ?></th>
+                  </tr>
+              </thead>
+
+              <tbody>
+                  <?php
+                  foreach ($logs as $log) :
+                      // Skip the index.html file.
+                      if ($log == 'index.html') {
+                          continue;
+                      }
+                      ?>
+                  <tr>
+                      <td class="column-check">
+                          <input type="checkbox" value="<?= $log; ?>" name="checked[]" />
```

Comment:
Anytime there could be user-supplied inputs we need to escape the values. In this case, something might have shown up in the logs that shouldn't be so let's be extra careful here. 

```suggestion
                          <input type="checkbox" value="<?= esc($log) ?>" name="checked[]" />
```